malibal/aon/s1/default.nix

```diff
@@ -0,0 +1,71 @@
+{ pkgs, lib, ... }:
+
+{
+  imports = [
+    ../../../common/pc/laptop
+    ../../../common/pc/laptop/ssd
+    ../../../common/cpu/intel
+    ../../../common/gpu/intel
+    ../../../common/gpu/nvidia/disable.nix
+  ];
+
+  hardware = {
+    enableAllFirmware = lib.mkDefault true;
```

Comment:
Which of the hardware files from here are you enabling? https://github.com/NixOS/nixpkgs/blob/797f7dc49e0bc7fab4b57c021cdf68f595e47841/nixos/modules/hardware/all-firmware.nix#L42
Please add this as a comment.

Reply:
I'm not exactly sure, so I guess I'm better leaving it off by now.

Between cpu, bluetooth, wifi and multiple controllers I guess everyone wants it, but I guess it's better to keep that explicit.